x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/index.tsx

```diff
@@ -259,16 +259,14 @@ export const AllCases = React.memo(() => {
             />
           </FlexItemDivider>
           <EuiFlexItem grow={false}>
-            <EuiButton fill href={getCreateCaseUrl()} iconType="plusInCircle">
```

Comment:
can we store results of `getCreateCaseUrl()` and `getConfigureCasesUrl()`, so we don't call them every time the component re-renders?

Reply:
Done!